manifest/docker_schema2.go

```diff
@@ -267,6 +267,7 @@ func (m *Schema2) Inspect(configGetter func(types.BlobInfo) ([]byte, error)) (*t
 	if err != nil {
 		return nil, err
 	}
+	layerInfos := m.LayerInfos()
```

Comment:
`LayerInfos` are computed after the `Unmarshal` in the other two implementations, this one differs for no obvious reason.